server/src/instant/reactive/session.clj

```diff
@@ -563,24 +589,27 @@
 
     (grouped-queue/inflight-queue-reserve 1 inflight-q)))
 
+(defn start-receive-worker [store-conn eph-store-atom receive-q stop-signal id]
```

Comment:
Extracted this into a function so that it would be easier to use inside of the session tests.